examples/AspNetCore/appsettings.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "AllowedHosts": "*",
+  "UseLogging": true,
```

Comment:
Not blocking - this name could be a bit misleading, `UseLogging` could mean:
1. Enable logging, send logs to the specified log exporter.
2. Enable logging, and convert logs to Activity events, export them via the specified trace exporter.

If we want the example to cover all three pillars (logs/metrics/traces), we probably need to revisit the configuration. This can be done later.

Reply:
Updated to the name @cijothomas proposed below.